roadmap.md

```diff
@@ -0,0 +1,44 @@
+# WebDriver BiDi roadmap
+
+## Real-world end-to-end user scenarios
+
+This document presents an overview of real-world end-to-end user scenarios we aim to enable via the WebDriver BiDi protocol. Each scenario requires one or more WebDriver BiDi commands and events to be specified, tested, and implemented across browser engines.
+
+### A. Extracting headlines
+
+This scenario loads a web page and uses script evaluation to extract the headlines on the page. In spec terms, this involves:
+
+- [x] [handling sessions](https://w3c.github.io/webdriver-bidi/#module-session)
+- [x] [creating a new `browsingContext`](https://w3c.github.io/webdriver-bidi/#command-browsingContext-create)
+- [x] [navigating to a URL](https://w3c.github.io/webdriver-bidi/#command-browsingContext-navigate)
+- [x] [evaluating JavaScript in the page context](https://w3c.github.io/webdriver-bidi/#command-script-evaluate)
```

Comment:
plot twist: you evaluate JavaScript to `document.querySelector` elements but the page author re-defined/deleted `document.querySelector` => the need for sandboxes

Reply:
I like that suggestion; sandboxes nicely fit into this e2e use case (or any of the following ones) as a requirement for robustness against mutated built-ins.